### PR TITLE
Replace underscores with hyphen in Colang 2 filenames and folders

### PR DESCRIPTION
**Pull request by schuellc-nvidia**
## Description

Replace underscores with hyphen in Colang 2 filenames and folders

### CODE REVIEW

File: tests/test_configs/multi_modal_demo_v2_x/demo.co
Comment (Pouyanpi) [inline tests/test_configs/multi_modal_demo_v2_x/demo.co:1]: just to be sure it is not accidentally here.

**Review by Pouyanpi — COMMENTED** [review]
Looks good, thanks! I'll build it and see if everything is OK.

File: tests/test_configs/multi_modal_demo_v2_x/demo.co
Comment (schuellc-nvidia) [reply tests/test_configs/multi_modal_demo_v2_x/demo.co:1]: Thanks for catching this! This changes were accidentaly committed!

**Review by Pouyanpi — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**